general/globals.hpp

```diff
@@ -90,7 +90,7 @@ std::string MakeParFilename(const std::string &prefix, const int myid,
     Functions for getting and setting the MPI communicator used by the library
     as the "global" communicator.
 
-    Currently, the MFEM "global" communicator is used only by the function
+    One place where the MFEM "global" communicator is used by the function
     mfem_error(), invoked when an error is detected - the "global" communicator
```

Comment:
This sentence is incomplete as is. Maybe change to: "The MFEM "global" communicator is used for example in the function..."?

Reply:
Done.